cli/exchange/pattern.go

```diff
@@ -224,10 +224,10 @@ func PatternPublish(org, userPw, jsonFilePath, keyFilePath, pubKeyFilePath, patN
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("the pattern definition (%s) must contain services, unable to proceed", patFile.Services))
 	}
 
+	keyVerified := false
```

Comment:
I dont see changes for the pattern verify function, I think we need similar key checks.